docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker in the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow this guide to register a cluster-wide [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) in the Service Catalog.
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
+* [helm](https://github.com/helm/helm#install)
+
+## Steps
+
+1. Clone the [`service-catalog`](https://github.com/kubernetes-incubator/service-catalog) repository to your local machine:
+    ```
+    git clone https://github.com/kubernetes-incubator/service-catalog.git
+    ```
+
+2.  Run this command to install the chart with the `ups-broker` name in the `ups-broker` Namespace:
+      ```
+     helm install service-catalog/charts/ups-broker --name ups-broker --namespace ups-broker
+     ```
+
+3. Register your broker:
+     ```
+    kubectl create -f service-catalog/contrib/examples/walkthrough/ups-broker.yaml
+    ```
+     After you successfully register your ServiceBroker or ClusterServiceBroker, the Service Catalog periodically fetches services from this broker and creates ServiceClasses or ClusterServiceClasses from them.
+
+4. Check the status of the broker:
+     ```
+    kubectl get clusterservicebrokers ups-broker -o jsonpath="{.status.conditions}"
+    ```
+     >**NOTE:** In case of ServiceBrokers, run:
```

Comment:
Lines 35 - 39 won't work in terms of formatting. 
Furthermore, I think the GSG should focus on the broker that you install in the previous step. Showing this option has no benefit at this point.

Reply:
This gsg should also be useful for users who want to register a ServiceBroker. That's why we need these notes.

Let's discuss the formatting issue - everything displays nicely on the view mode so I do not know what you mean.